res/xml/qwertz_cs_minimal.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<keyboard>
+  <row>
+    <key                 key0="q"         key1="1"         key2="esc"       key3="+"          key4="~"      />
```

Comment:
Keys on the edge of the screen (`key1` and `key3` here) are harder to type. It would be best to move these elsewhere. Some keys from the first row could be pushed down to the second row as there are some free spots.

On qwerty, this was solved by moving the digit close to the edge of the screen to the opposite position (inconsistent with the other digits). The escape key is on `key4`.